code/modules/mob/mob_helpers.dm

```diff
@@ -326,22 +326,17 @@ It's fairly easy to fix if dealing with single letters but not so much with comp
 #define PIXELS_PER_STRENGTH_VAL 16
 
 /proc/shake_camera(mob/M, duration, strength=1)
-	set waitfor = 0
-	if(!M || !M.client || M.shakecamera || M.stat || isEye(M) || isAI(M))
+	var/current_time = world.time
```

Comment:
Why do you store this?

Reply:
I was just thinking it would be faster than getting it twice... is it faster who knows?